scripts/circleci/run_devtools_tests_for_react_versions.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const {exec} = require('child-process-promise');
+const chalk = require('chalk');
+const {join} = require('path');
+
+const ROOT_PATH = join(__dirname, '..', '..');
+
+const tmpBuildPath = join(ROOT_PATH, 'build-tmp');
+const buildPath = join(ROOT_PATH, 'build');
+
+const version = process.argv[2] || '17.0.0';
+
+// function getReactVersions() {
+//       const downloadResult = exec(
+//         `npm install --prefix ${tmpBuildPath}` +
+//           ` react-dom@${version} react@${version}`
+//       );
+//       const versionResult = await exec('yarn info react versions');
+
+//       if (versionResult.stderr) {
+//         console.error(
+//           chalk.red(
+//             `yarn info react versions failed wtih the following error: ${versionResult.stderr}`
+//           )
+//         );
+
+//     const versionsData = getReactVersions(versionResult.stdout.toString());
+
+//   const dataStart = versionData.indexOf('[') + 1;
+//   const dataEnd = versionData.indexOf(']');
+//   return versionData
+//     .slice(dataStart, dataEnd)
+//     .split(',')
+//     .map(line => line.trim().replaceAll("'", ''))
+//     .filter(line => !line.includes('-') && +line.split('.')[0] >= +minVersion);
+// }
+
+async function runTests() {
+  console.log(chalk.bold(`Downloading React v${version}`));
+  await exec(`mkdir ${tmpBuildPath}-${version}`);
+  await exec(
+    `npm install --prefix ${tmpBuildPath}-${version}` +
+      ` react-dom@${version} react@${version}`
+  );
+
+  await exec(
```

Comment:
This will install and test all the versions sequentially which will be pretty slow. You can take advantage of CircleCI's `matrix` feature to split it across multiple parallel CI jobs. That's what we do for the regular test suite:

https://github.com/facebook/react/blob/069d23bb74939c19bc8860ad4013324bd6e1c5f3/.circleci/config.yml#L381-L405

Reply:
I was trying to figure out how to do this by dynamically getting the versions from NPM inside the circleci config, but I don't think there's a good way to do it. 😭

Reply:
Do you really need to get them dynamically? I would just hard code it to the versions we care about